website/cue/reference/remap/functions/to_timestamp.cue

```diff
@@ -11,18 +11,20 @@ remap: functions: to_timestamp: {
 			name:        "value"
 			description: "The value that is to be converted to a timestamp. If a string, must be a valid representation of a `timestamp`, and no `default` exists, an `ArgumentError` will be raised."
 			required:    true
-			type: ["string", "integer", "timestamp"]
+			type: ["string", "float", "integer", "timestamp"]
 		},
 	]
 	internal_failure_reasons: [
 		"When `value` is a `string`, it is not a valid timestamp format",
 		"When `value` is an `int`, it is not within the Unix timestamp range",
+		"When `value` is an `float`, it is not within the Unix timestamp range",
```

Comment:
```suggestion
		"When `value` is a `float`, it is not within the Unix timestamp range",
```

Reply:
👍 thanks. Fixed in 47c77c39b

Reply:
😛 got it in one place, but not the other